common/icons/archive.js

```diff
@@ -0,0 +1,13 @@
+const Archive = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <g data-name="Layer 2">
```

Comment:
We have (I think) tried to put a class of `icon__shape` on each piece of an svg so far, so we can change/inherit colour in CSS. Did we have an `svgmin` task that got rid of all the extraneous attributes as well somewhere?